Demos/Eyes/EyesWidget.cpp

```diff
@@ -65,11 +65,12 @@ void EyesWidget::render_eyeball(int index, GUI::Painter& painter) const
 {
     auto eye_width = width() / m_num_eyes;
     Gfx::Rect bounds { index * eye_width, 0, eye_width, height() };
-    auto thickness = max((width() + height()) / 15, 1);
+    auto height_thickness = max(int(height() / 5.5), 1);
+    auto width_thickness = max(int(eye_width / 5.5), 1);
```

Comment:
Just one tiny nit: let's put width before height. (Swap the two above lines.)

Reply:
Okies.